universal-application-tool-0.0.1/app/views/admin/programs/ProgramFormBuilder.java

```diff
@@ -63,6 +66,13 @@ private static ContainerTag buildProgramForm(
             .setLabelText("Describe this program for administrative use")
             .setValue(adminDescription)
             .getContainer(),
+        FieldWithLabel.checkbox()
+            .setId("program-hide-from-view-checkbox")
+            .setFieldName("hideFromView")
+            .setLabelText("Hide this program")
```

Comment:
suggestion (blocking): "hide" here could be a bit ambiguous since the program is still accessible via URL. recommend changing the label text to `hide this program on public index page`

Reply:
i tried to think of other things that could go here but they're all kinda wordy, unfortunately. on the plus side, the admin will definitely be on the laptop and there is plenty of lateral space here in the UI

Reply:
Good point, done

<img width="1427" alt="Screen Shot 2021-09-13 at 3 27 37 PM" src="https://user-images.githubusercontent.com/66492295/133164626-17de916a-03ee-4b2e-b2f0-8e47a16cebec.png">